essay.tex

```diff
@@ -27,19 +27,32 @@
 
 \section{Introduction}
 
-Write your introduction here. A brief introduction is recommended, which should outline key details of the chosen topic and the reviewed papers, motivate the work, and provide a roadmap of key points to the reader. The motivation is quite important here, as essays should have a contribution (i.e., what is the point of the essay, and what does the reader take away from the essay) and the link between the motivation (in the introduction) and the contribution (in the conclusion) should be made clear.
+In this essay I will outline the key benefits and drawbacks of daily scrum meetings and using my knowledge gained from research to discuss how they could be leveraged or overcome to make daily scrum meetings as effective as possible. I will first outline what a daily scrum meeting is and why it is important and useful, I will then discuss the benefits, drawbacks, and solutions to those drawbacks of efficient daily scrum meetings found in my research and give my own conclusions, followed by the final conclusion. In this paper I hope to provide solutions to the most common obstacles to having an efficient daily meeting and highlight good practice in a daily meetings. After reading this you should know how to make the most out of daily scrum meetings and be as productive as possible.
 
-\section{Your section title here}
+\section{Daily Scrum}
 
-Write the main body of your essay here. Add more sections if appropriate. You may choose to write about each of your three papers in its own section, or you may choose a different structure. Either way, remember that you are being assessed on technical insight and analysis: it is not enough to merely summarise the contents of the three papers. You must demonstrate the ability to make inferences beyond what is written in the papers, and to draw the three papers together into a single coherent narrative.
+As the agile manifesto \cite{Manifesto} states ``The most efficient and effective method of 
+conveying information to and within a development 
+team is face-to-face conversation''. A daily scrum meeting is a daily face to face 15-minute meeting, allowing the members of a team to inspect the progress of work and to remove impediments. It is established by the Scrum master and takes place each day at the same place and time, thus avoiding unnecessary coordination. Each team member answers the following three questions: (1) What have you worked on since the last meeting? (2) What will you work on until the next meeting? (3) Have any impediments occurred? \cite{ScrumEveryDay}. It is the Scrum masters responsibility to resolve any impediments that arise during the meeting. Team members meet in front of a task board and update their status of work \cite{Multitouch}, allowing team to check the status of sprint backlog items on a daily basis and thus keeps the team focused \cite{ScrumEveryDay}.
```

Comment:
Elaborate on this to argue *why* it's an important activity in game development.